contracts/RelayHub.sol

```diff
@@ -123,6 +124,9 @@ contract RelayHub is IRelayHub {
     returns (IPaymaster.GasLimits memory gasLimits, uint256 maxPossibleGas) {
         gasLimits =
             IPaymaster(relayRequest.relayData.paymaster).getGasLimits();
+
+        require(paymasterMaxAcceptanceBudget > gasLimits.acceptanceBudget, "unexpected high acceptanceBudget");
```

Comment:
Isn't it supposed to be `>=`? No test for that?

Reply:
you're right. I should be >=